github/repos_codeowners_test.go

```diff
@@ -62,12 +62,74 @@ func TestRepositoriesService_GetCodeownersErrors(t *testing.T) {
 
 	const methodName = "GetCodeownersErrors"
 	testBadOptions(t, methodName, func() (err error) {
-		_, _, err = client.Repositories.GetCodeownersErrors(ctx, "\n", "\n")
+		_, _, err = client.Repositories.GetCodeownersErrors(ctx, "\n", "\n", nil)
 		return err
 	})
 
 	testNewRequestAndDoFailure(t, methodName, client, func() (*Response, error) {
-		got, resp, err := client.Repositories.GetCodeownersErrors(ctx, "o", "r")
+		got, resp, err := client.Repositories.GetCodeownersErrors(ctx, "o", "r", nil)
+		if got != nil {
+			t.Errorf("testNewRequestAndDoFailure %v = %#v, want nil", methodName, got)
+		}
+		return resp, err
+	})
+}
+
+func TestRepositoriesService_GetCodeownersErrors_specificRef(t *testing.T) {
+	client, mux, _, teardown := setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/codeowners/errors", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		testHeader(t, r, "Accept", mediaTypeV3)
```

Comment:
Please add `testFormValues` to this test.
For example, please see: https://github.com/google/go-github/blob/master/github/code-scanning_test.go#L139

Reply:
Thanks, done!